CONTRIBUTING.md

```diff
@@ -17,12 +17,36 @@ want a `.war` as quick as possible, you can use this instead:
 $ mvn -Dmaven.test.skip=true clean package 
 ```
 
+
 # Suggesting modifications
 
 Airsonic's source code is hosted on [github](https://github.com/airsonic/airsonic/),
 who provides a [lot of documentation](https://help.github.com/en) on how
 to contribute to projects hosted there.
 
+Keep in mind that we are a small project, and that we value stability over new
+features.
+
+
+# Getting your pull-requests reviewed an merged
+
+Once you issued a pull-request, it will be reviewed by other contributors, and
+modifications/enhancement could be suggested. Once all the concerns are
+addressed, it might get merged, which usually requires two [core
+contributors](https://github.com/airsonic/airsonic/graphs/contributors): one to
+explicitly approve the pull-request, and a second to do the actual merge.
+Note that incremental/small changes are likely going to get reviewed, accepted
+and merged quicker than huge ones.
```

Comment:
Do we need a note in here like: "Keep in mind that we strive to balance stability with new features while best utilizing our limited time. As such, your Pull Request might be rejected when it doesn't strike that balance."

Reply:
Or we could just have better build practices?

Release stability should really not be mixed with branch stability or usability.

If we have that much of a concern about keeping master stable, then we should provide additional branches for people to merge commits to, and merge in those branches into master after releases.

Release cycles should be separated and less tightly coupled to branch commits.

Reply:
I don't get why you prefer to have pull-requests in branches, instead of having, … well, pull-requests. What would the advantages be?

Reply:
I think the commit granularity can be set flexibly.
This level of expression may be fine :

```If the PR is known to be very large, the committer can check with the maintainer in advance about the degree of division of the PR.```

I think the search update was this way (I think..).
The maintainer can takes the lead in sizing, gives advice, and reduce friction.

Large and small can become subjective, so it may be desirable to resolve them before accepting them.